io.github.zzzgydi.ClashVerge.yml

```diff
@@ -0,0 +1,79 @@
+app-id: io.github.zzzgydi.ClashVerge
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: clash-verge
+rename-icon: clash-verge
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.config/clash-verge
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.login1
```

Comment:
Why?

Reply:
Because these allow Clash Verge to retain configuration files, running in the background can display them in the system tray

Reply:
That's not what `org.freedesktop.login1` is for.

Reply:
> That's not what `org.freedesktop.login1` is for.

Indeed